### ISSUE TITLE
Request - cleanup request - ctlithostratigraphic_bed

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#9595]
Help us understand your request (check below):

- [x] other

The data in ctlithostratigraphic_bed are pretty rough, help please.

There's a partially-cleaned spreadsheet here: https://docs.google.com/spreadsheets/d/1f--l9nEyHWjaLlt79lI_VCoPZsCHoVt5HNW_cDgHfPY/edit?gid=225044792#gid=225044792

Anyone can view, let me know if you want edit access.

I moved some stuff around but didn't check any links, try to write descriptions, etc.

HELP!

------------

Please review the [Arctos Code of Conduct](https://arctosdb.org/code-of-conduct/) to ensure respectful communication, and see <https://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html> for more information about submitting and managing Issues.

**Comment by ekrimmel** [comment]
FYI @ArctosDB/paleontology has this in queue to review. Won't happen particularly fast but it is on the radar. That [working spreadsheet is here](https://docs.google.com/spreadsheets/d/1VVsKQLPa95VE6ybZUT9S3qUrdr5k_QUq6kaVjZIsEVc/edit?gid=774958859#gid=774958859).

**Comment by WaigePilson** [comment]
Yes @dustymc there are several issues related to lithostratigraphy code table clean up, but all of them should be redirected here: https://github.com/ArctosDB/arctos/issues/9249 (see the linked working spreadsheet, also linked above) as that is the active issue we are working on!! Don't want you to re-do a lot of our work by pulling these various code tables and cleaning them.

**Comment by dustymc** [comment]
@WaigePilson if you've got it covered then this issue isn't necessary, closing, thanks!
